src/pair.h

```diff
@@ -18,6 +18,20 @@
 
 namespace LAMMPS_NS {
 
+enum {
+  ENERGY_NO_COMPUTATION  = 0x00,
```

Comment:
some of those constants are rather long names. not sure that helps with readability.
how about this?
- ENERGY_NONE
- ENERGY_GLOBAL
- ENERGY_PERATOM
- VIRIAL_TALLY
- VIRIAL_FDOTR
- VIRIAL_PERATOM
- VIRIAL_CENTROID

Reply:
@akohlmey sure, that seems ok